webapp/src/test/java/com/box/l10n/mojito/quartz/multi/QuartzMultiSchedulerConfigurationPropertiesTest.java

```diff
@@ -0,0 +1,73 @@
+package com.box.l10n.mojito.quartz.multi;
+
+import static org.junit.Assert.assertTrue;
+
+import com.box.l10n.mojito.quartz.QuartzSchedulerManager;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(
+    classes = {
+      QuartzMultiSchedulerConfigurationProperties.class,
+      QuartzMultiSchedulerConfig.class,
+      QuartzMultiSchedulerManager.class,
+      QuartzMultiSchedulerConfigurationPropertiesTest.class
+    },
+    properties = {
+      "l10n.org.multi-quartz.enabled=true",
+      "l10n.org.multi-quartz.schedulers.default.quartz.threadPool.threadCount=10",
+      "l10n.org.multi-quartz.schedulers.lowPriority.quartz.threadPool.threadCount=5"
+    })
+@EnableConfigurationProperties(QuartzMultiSchedulerConfigurationProperties.class)
+public class QuartzMultiSchedulerConfigurationPropertiesTest {
```

Comment:
could you keep that test a pure unit test for QuartzMultiSchedulerConfigurationProperties and have another test for  QuartzSchedulerManager. (as i tried to test the mapping quickly, the test barked at me because of the schedulerManager)